samples/apps/autogen-assistant/README.md

```diff
@@ -0,0 +1,96 @@
+# AutoGen Assistant
+
+![ARA](./docs/ara_stockprices.png)
+
+AutoGen Assistant is an Autogen-powered AI app (user interface) that can converse with you to help you conduct research, write and execute code, run saved skills, learn new skills (explicitly and by demonstration), and adapt in response to your interactions.
+
+Project Structure:
+
+- _autogenra/_ code for the backend classes and web api (FastAPI)
+- _frontend/_ code for the webui, built with Gatsby and Tailwind
+
+## Getting Started
+
+AutoGen requires access to an LLM. Please see the [AutoGen docs](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints) on how to configure access to your LLM provider. In this sample, We recommend setting up your `OPENAI_API_KEY` environment variable and then specifying the exact model names to be used in the `llm_config` that is passed to each agent specification. See the `get_default_agent_config()` method in `utils.py` to see an example of setting up `llm_config`.
```

Comment:
This recommendation is for OpenAI only. Are we sure about not recommending a way to use Azure OpenAI?

Reply:
Also trying to run this on Azure OpenAI and failing.

Reply:
@sonichi, yes, `llm_config` should support any model that can be specified with AutoGen.  

```python
 llm_config = LLMConfig( 
        config_list=[{"model": "gpt-4"}], # this could point to azure openai or any other compliant model
        temperature=0,
    )
```
@ArneJanning ... how did you try using Azure OpenAI? What was the failure? Without this description, it is hard to fix the issue.